UM/VersionUpgradeManager.py

```diff
@@ -142,22 +144,22 @@ def registerCurrentVersion(self, version_info: Tuple[str, int], type_info: Any)
     #   The upgrade plug-ins must all be loaded at this point, or no upgrades
     #   can be performed.
     #
-    #   \return True if anything was upgraded, or False if it was already up to
-    #   date.
-    def upgrade(self) -> bool:
+    def upgrade(self):
```

Comment:
```suggestion
    def upgrade(self) -> None:
```
Mypy doesn't check the function otherwise.